packages/server/__tests__/api/grants.test.js

```diff
@@ -324,6 +324,62 @@ describe('`/api/grants` endpoint', () => {
             });
         });
     });
+    context('DELETE /api/grants/:grantId/interested/:agencyId', () => {
+        context('by an admin user', () => {
+            const interestEndpoint = `335255/interested`;
+            it('allows removing grant interest for a single agency', async () => {
+                const response = await fetchApi(`/grants/${interestEndpoint}/undefined`, agencies.own, {
+                    ...fetchOptions.admin,
+                    method: 'delete',
+                    body: JSON.stringify({ agencyIds: [agencies.own], interestedCode: null }),
+                });
+                expect(response.statusText).to.equal('OK');
+            });
+            it('allows removing grant interest for multiple authorized agencies', async () => {
+                const response = await fetchApi(`/grants/${interestEndpoint}/undefined`, agencies.own, {
+                    ...fetchOptions.staff,
+                    method: 'delete',
+                    body: JSON.stringify({ agencyIds: [agencies.own, agencies.ownSub], interestedCode: null }),
+                });
+                expect(response.statusText).to.equal('OK');
```

Comment:
should we be asserting on the actual database state after the call here, not just the HTTP response? (and same for the other cases here)

Reply:
I like where you're going with this, but I think it's worth first deciding what purpose/proof we think these tests provide. Right now, they come across like integration tests more than unit tests, and I don't think integration tests should need to check DB state. 

NB: I'd prefer to have unit tests without DB connectivity as well, but I think that's a larger effort beyond the scope of this PR.